lib/Marlin/Marlin/src/module/configuration_store.cpp

```diff
@@ -147,8 +147,8 @@ static float get_steps_per_unit(size_t index) {
 }
 #else
 static constexpr float get_steps_per_unit(size_t index) {
-  static constexpr float _DASU[] = DEFAULT_AXIS_STEPS_PER_UNIT;
-  return pgm_read_float(&_DASU[ALIM(i, _DASU)]);
+  constexpr float _DASU[] = DEFAULT_AXIS_STEPS_PER_UNIT;
+  return pgm_read_float(&_DASU[ALIM(index, _DASU)]);
```

Comment:
how could the previous code even compile?
`pgm_read_float(&_DASU[ALIM(i, _DASU)]);`

Reply:
it did not, because of preprocessor